src/components/species-modal/species-modal-component.jsx

```diff
@@ -117,8 +127,13 @@ const SpeciesModalComponent = ({
             />
           </div>
           <div className={styles.summary}>
-            {speciesNumber()} LAND
-            VERTEBRATE SPECIES
+            <Tabs
```

Comment:
I completely forgot to tell you could you hide this if we don't have the REACT_APP_FEATURE_MARINE feature tag active?